4.analyze_data/notebooks/UMAP/scripts/1.vis_UMAP.r

```diff
@@ -139,11 +141,46 @@ umap_siRNA_construct_gg <- (
     + scale_color_gradient(
             name = "Dose (nM)",
             low = "#feaaa3", high = "#ee2711",
-            na.value = "#727272"  # Set color for "none" facet to grey
+            na.value = "#727272"
         )
     + facet_wrap(~ Metadata_siRNA, drop = FALSE)
 )
 
 ggsave(output_file, umap_siRNA_construct_gg, dpi = 500, height = 6, width = 6)
 
 
+
+# Select concat plate file path from list of umap files
+concat_plate_path <- umap_files[[1]]
+
+# Load in the umap data for plate 4 only
+df <- readr::read_tsv(
+    concat_plate_path,
+    col_types = readr::cols(
+        .default = "d",
+        "Metadata_Plate" = "c",
+        "Metadata_Well" = "c",
+        "Metadata_Site" = "c",
+        "Metadata_number_of_singlecells" = "c",
+        "Metadata_genotype" = "c"
+    )
+)
+
+# Plate UMAP
+output_file <- "./figures/UMAP_Concat_plate.png"
+
+# UMAP labeled with plate
+umap_plate_gg <- (
+    ggplot(df, aes(x = UMAP0, y = UMAP1))
+    + geom_point(
+            aes(color = Metadata_Plate), size = 1.2, alpha = 0.5
+    )
+    + theme_bw()
+    + scale_color_manual(
+        name = "Plate",
+        values = c("Plate_3" = "#7570b3", "Plate_3_prime" = "#e7298a", "Plate_5" = "#d95f02")
+    )
+)
```

Comment:
Note this is a stylistic comment and feel free to ignore as this is a personal preference.

I tend to write my R code more pythonically. See example below and feel free to accept or reject!

```suggestion
umap_plate_gg <- (
    ggplot(
    	df, 
    	aes(
    		x = UMAP0, 
    		y = UMAP1
		)
	)
    + geom_point(
        aes(
	       	color = Metadata_Plate
        ), 
        size = 1.2, alpha = 0.5
    )
    + theme_bw()
    + scale_color_manual(
        name = "Plate",
        values = c(
        	"Plate_3" = "#7570b3", 
        	"Plate_3_prime" = "#e7298a", 
        	"Plate_5" = "#d95f02"
    	)
    )
)
```

Reply:
This is a fair comment! Do you happen to know of a linter that already does this or do you have to manually do this as you write the R code?